app/lib/sdk/orbisDB/context.tsx

```
@@ -0,0 +1,202 @@
+import { title } from 'process';
+
+interface OrbisContextProps {
+  authResult: OrbisConnectResult | null;
+  setAuthResult: React.Dispatch<
+    React.SetStateAction<OrbisConnectResult | null>
+  >;
+  insert: (value: any, modelId: string) => Promise<void>;
+  update: (docId: string, updates: any) => Promise<void>;
+  getAssetMetadata: (assetId: string) => Promise<any>;
+  orbisLogin: (privateKey?: string) => Promise<OrbisConnectResult>;
+  isConnected: (address: string) => Promise<Boolean>;
```

Biome

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)